cmd/ctrlc/root/run/exec/runner.go

```diff
@@ -25,6 +27,10 @@ type ExecConfig struct {
 }
 
 func (r *ExecRunner) Status(job api.Job) (api.JobStatus, string) {
```

Comment:
if the pid is missing is that considered sucessful? what if the pid is now a different process?

Reply:
its not clear how to keep track of these processes, maybe we assum they are not deamon so if the command closes so does the children. This means we won't need this status check and instead create a goroutine that updates the job when it exists